SoC-2016-Ideas.md

```diff
@@ -46,20 +46,34 @@ your microproject in your application (e.g. linking to
 participate in the review of other patches, then you may also include
 links to discussions that would support your application.
 
-Students must send drafts of their proposal on
-the mailing-list before submitting it officially to GSoC to get
-feedback from the community. They are strongly encourraged to publish a
-draft on the official GSoC website *and* post it to the mailing list
-for discussion.
+Students must send drafts of their proposal on the mailing-list before
+submitting it officially to GSoC to get feedback from the
+community. They are strongly encourraged to publish a draft on the
+official GSoC website *and* post it to the mailing list for
+discussion.
+
+Git development is based on sending successive versions of patchs or
+patch series to the mailing list until they are considered good and
+correct by the reviewers and Junio Hamano, the maintainer, who will
+merge them. This process usually takes quite some time. By sending
+drafts of your proposal, and patches for your microproject, to the
+mailing list long before the deadline, you can show us that you are
+willing and able to work well using the Git development process.
+
+It is *expected* that what you send will need several rounds of
+reviews and discussions. If you not sure at all about a patch you can
+put "[RFC/PATCH]" at the beginning of its subject. If you are not sure
```

Comment:
This is partly redundant with the microproject page which already talks about the flow, but provides useful additions. I'm wondering if we shouldn't instead add the above to the microproject page (near the link to SubmittingPatches) and write something like this here:

"Just like for submitting patches (see the explanations in the [microproject](SoC-2016-Microprojects) page), your proposal is likely to trigger a lot of discussion, and it is expected that you send multiple iterations of your proposal."